lib/BridgedRoom.js

```diff
@@ -264,39 +265,81 @@ BridgedRoom.prototype._handleSlackMessage = function(message, ghost) {
     else if (message.files) {
         for (var i = 0; i < message.files.length; i++) {
             const file = message.files[i];
-            // We also need to upload the thumbnail
-            let thumbnail_promise = Promise.resolve();
-            // Slack ain't a believer in consistency.
-            const thumb_uri = file.thumb_video || file.thumb_360;
-            if (thumb_uri) {
-                thumbnail_promise = ghost.uploadContentFromURI(
-                    {
-                        // Yes, we hardcode jpeg. Slack always use em.
-                        title: `${file.name}_thumb.jpeg`,
-                        mimetype: "image/jpeg",
-                    },
-                    thumb_uri,
-                    this._slack_bot_token
-                );
+            if (file.mode === "snippet") {
+                return rp({
+                    uri: file.url_private,
+                    headers: {
+                        Authorization: `Bearer ${this._slack_bot_token}`,
+                    }
+                }).then(function (htmlString) {
+                    var code = '```';
+                    code += '\n';
+                    code += htmlString;
+                    code += '\n';
+                    code += '```';
+                    if (file.filetype) {
+                        var html_code = '<pre><code class="language-' + file.filetype + '">';
```

Comment:
define ``let htmlCode`` above, and then set it in these. Also please don't use snake case.